hera_mc/mc_session.py

```diff
@@ -748,6 +748,81 @@ def get_paper_temps(self, starttime, stoptime=None):
         return self._time_filter(PaperTemperatures, 'time', starttime,
                                  stoptime=stoptime)
 
+    def add_weather_data(self, time, variable, value):
+        """
+        Add new weather data to the M&C database.
+
+        Parameters:
+        ------------
+        time: astropy time object
+            astropy time object based on a timestamp from the katportal sensor.
+        variable: string
+            must be a key in weather.weather_sensor_dict
+        value: float
+            value from the sensor associated with the variable
+        """
+        from .weather import WeatherData
+
+        self.add(WeatherData.create(time, variable, value))
+
+    def add_weather_data_from_sensors(self, starttime, stoptime, variables=None):
+        """
+        Add weather data for a given variable and timespan from KAT sensors.
```

Comment:
This function connects to the meerkat db and grabs the latest data using the "create_from_sensors" function.

Reply:
ok, I'll add that